### PR TITLE
Add label and loss support for BERT/RoBERTa/ERNIE

### PR DESCRIPTION
**Pull request by guoshengCS**
### PR types

New features
### PR changes

APIs
### Description

Add label and loss support for BERT/RoBERTa/ERNIE

### CODE REVIEW

File: docs/source/paddlenlp.taskflow.rst [hunk 1]
```diff
@@ -28,6 +29,7 @@ paddlenlp.taskflow
    paddlenlp.taskflow.sentiment_analysis
    paddlenlp.taskflow.task
    paddlenlp.taskflow.taskflow
+   paddlenlp.taskflow.text2image_generation
```
Comment (ZeyuChen) [inline docs/source/paddlenlp.taskflow.rst:32]: text_to_image?

**Review by ZeyuChen — COMMENTED** [review]
text2image_generation -> text_to_image

File: docs/source/paddlenlp.taskflow.rst (hunk 1 above)
Comment (guoshengCS) [reply docs/source/paddlenlp.taskflow.rst:32]: OK， 这个还请 @JunnYu 处理下，涉及文件名、Taskflow task名和文档

**Review by yingyibiao — APPROVED** [review]
LGTM

**Pull request merged**